docs/reference/mapping/types/date.asciidoc

```diff
@@ -91,6 +91,30 @@ PUT my-index-000001
 }
 --------------------------------------------------
 
+
+[[missing-date-components]]
+==== Missing date components
+
+For <<query-dsl-range-query,range>> queries and
+<<search-aggregations-bucket-daterange-aggregation,date range>> aggregations,
+{es} replaces missing date components with the following values. Missing year
+components are not replaced.
+
+[source,text]
+----
+MONTH_OF_YEAR:    01
+DAY_OF_MONTH:     01
+HOUR_OF_DAY:      23
+MINUTE_OF_HOUR:   59
+SECOND_OF_MINUTE: 59
+NANO_OF_SECOND:   999_999_999
+----
```

Comment:
Are these values only applied to `gt` or `gte` values or do they apply to `lt` and `lte` as well?